modules/apps/27-interchain-accounts/types/metadata.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	"github.com/cosmos/ibc-go/v7/internal/collections"
```

Comment:
In `03-connection`, this is imported as `collections "github.com/cosmos/ibc-go/v7/internal/collections"`. I slightly prefer no alias, so it'd be nice if you could also change this in `03-connection`